gateway-service-admin/src/main/java/org/apache/knox/gateway/service/admin/AliasResource.java

```diff
@@ -100,15 +105,20 @@ public Response postAlias(@PathParam("topology") final String topology,
     final AliasService as = getAliasService();
 
     try {
+      /* check if the string is properly encoded and expect encoded values */
+      topology = URLDecoder.decode(topology, StandardCharsets.UTF_8.name());
+      alias = URLDecoder.decode(alias, StandardCharsets.UTF_8.name());
+      value = URLDecoder.decode(value, StandardCharsets.UTF_8.name());
```

Comment:
Does this limit the values that can be used for an alias?

Reply:
values ? I did not follow the comment. I will be taking out this piece of code because of your earlier comment.

Reply:
Yea sorry wasn't clear. I meant does the url decoding mess up what alias values can be stored. ie: `kevin%20password` but guess it doesn't matter since this gets removed anyway :)

Reply:
Ahh, I see, righto, I thought I was doing it wrong by not decoding it and including the 'application/x-www-form-urlencoded' but I was wrong.